index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
+        <a href=""><img src="images/Union.png" alt="" /></a>
+      </nav>
+    </header>
+    <div class="heading-section">
+      <div class="heading1">
+        <h1 class="heading">Hey There <br />I’m Leslie</h1>
+      </div>
+      <div class="describe">
+        <h2 class="describe">I am a Software Developer</h2>
```

Comment:
- [ ] I would suggest you change this to a paragraph tag i.e `<p>`. This way will be semantically correct and the screen readers and other technologies will thank you.